asv/environment.py

```diff
@@ -428,7 +428,7 @@ def get_environment_class(conf, python):
         if cls.matches_python_fallback and cls.matches(python):
             return cls
     raise EnvironmentUnavailable(
-        "No way to create environment for python='{0}'".format(python))
+        "No way to create environment for python='{python}'")
```

Comment:
does this need an `f`? You could use `flynt` or something to do these automatically (maybe out of scope for this PR)

Reply:
Yup, good eye. Also I'll open another PR for `flynt` but I think @jarrodmillman has converted a lot of it already.